src/main/resources/com/chikli/hudson/plugin/naginator/NaginatorAction/badge.jelly

```diff
@@ -1,2 +1,4 @@
 <?jelly escape-by-default='true'?>
-<img src="${resURL}/plugin/naginator/redo.png" title="${%Re-scheduled after failure}"/>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <l:icon src="symbol-arrow-redo-outline plugin-ionicons-api" class="icon-sm"/>
```

Comment:
I want to preserve `title`. We can use `tooltip` for `<l:icon>`.
Would you see that works?

Reply:
done